app/views/personaldetails/PersonalDetailsViewV2.scala.html

```diff
@@ -0,0 +1,147 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.ConfigDecorator
+@import controllers.auth.requests.UserRequest
+@import uk.gov.hmrc.renderer.TemplateRenderer
+@import scala.concurrent.ExecutionContext
+@import viewmodels.{PersonalDetailsTableRowModel, PersonalDetailsViewModel}
+
+@this(mainView: MainView)
+@(personalDetails: Seq[PersonalDetailsTableRowModel], trustedHelpers: Option[PersonalDetailsTableRowModel], paperlessHelpers: Option[PersonalDetailsTableRowModel], signinDetailsHelpers: Option[PersonalDetailsTableRowModel])(implicit request: UserRequest[_], configDecorator: ConfigDecorator, templateRenderer: TemplateRenderer, messages: play.api.i18n.Messages, ec: ExecutionContext)
+
+
+
+@mainView(messages("label.your_personal_details"),Some(messages("label.your_personal_tax_account"))) {
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <header class="page-heading">
+        <h1 class="heading-xlarge">
+          @request.name.map(upperCaseToTitleCase).getOrElse(messages("label.your_account"))
+        </h1>
+        <p class="heading-secondary"><span class="visuallyhidden">@messages("label.this.section.is") </span>@messages("label.your_profile")</p>
+      </header>
+
+    </div>
```

Comment:
lines 40 to 45 are not needed because you are needlessly declaring `column-two-thirds` twice when you can simply declare it once for the content.